### DISCUSSION TITLE
Wie Statusmeldungen auslesen?

### DISCUSSION THREAD

**Discussion by salzk** [open3e/open3e#128] [Q&A]
Hallo,

Danke für das geniale Projekt, konnte es dank der umfassenden Hilfe auf einem alten PI2 mit readonly Filesystem problemlos zum Laufen bringen. Jetzt eine Frage:

Gibt es die Möglichkeit die Statusmeldungen (wie sie z.B. in viGuide angezeigt werden) auszulesen?

LG
Reinhard

**Comment by Philip-Wiege** [comment]
Hi Reinhardt,

im DID Bereich von 257-266 gibt es die Fehler, Warnungs und Info-Listen. Zumindest im MQTT Explorer kann ich diese so auslesen. In HomeAssistant konnte ich diese noch nicht schön weiterreichen.

Grüße
Philip

**Reply by salzk** [reply]
Hi Philip,
Danke für die schnelle Antwort. Habe es schon erfolgreich getestet und werde mir mal überlegen, wie man das am Besten in HA verarbeitet.
LG
Reinhard

**Reply by Philip-Wiege** [reply]
Das "dumme" ist, dass Home Assistant leider kein Konzept von Listen oder Arrays etc. versteht. Mir ist kein Weg bekannt, wie man direkt alle Listeneinteäge auf einmal in einer Variable speichern kann ohne für jeden Eintrag eine eigene Entität anlegen zu müssen...
